[2025-04-22T16:37:57.767+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-22T16:37:57.777+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data scheduled__2025-04-22T15:00:00+00:00 [queued]>
[2025-04-22T16:37:57.782+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data scheduled__2025-04-22T15:00:00+00:00 [queued]>
[2025-04-22T16:37:57.782+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-22T16:37:57.789+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): meteo_operations.collect_meteo_data> on 2025-04-22 15:00:00+00:00
[2025-04-22T16:37:57.793+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'weather_data_pipeline', 'meteo_operations.collect_meteo_data', 'scheduled__2025-04-22T15:00:00+00:00', '--job-id', '215', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpjn4rzo4z']
[2025-04-22T16:37:57.795+0000] {standard_task_runner.py:105} INFO - Job 215: Subtask meteo_operations.collect_meteo_data
[2025-04-22T16:37:57.796+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=90) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-22T16:37:57.797+0000] {standard_task_runner.py:72} INFO - Started process 112 to run task
[2025-04-22T16:37:57.821+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data scheduled__2025-04-22T15:00:00+00:00 [running]> on host 8e2c8626a9ef
[2025-04-22T16:37:57.867+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='meteo_operations.collect_meteo_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-22T15:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-22T15:00:00+00:00'
[2025-04-22T16:37:57.868+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-22T16:37:57.889+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': 21.4901, 'longitude': 39.1862, 'timestamp': '2025-04-22 00:00:00', 'temperature': 27.073501586914062, 'humidity': 77.0, 'wind_speed': 3.096837043762207, 'pm2_5': 108.30000305175781, 'co': 441.0}
[2025-04-22T16:37:57.902+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-22T16:37:57.902+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=meteo_operations.collect_meteo_data, run_id=scheduled__2025-04-22T15:00:00+00:00, execution_date=20250422T150000, start_date=20250422T163757, end_date=20250422T163757
[2025-04-22T16:37:57.930+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-22T16:37:57.945+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-22T16:37:57.947+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
